kiosk/src/Models/Kiosk.ts

```diff
@@ -6,6 +6,15 @@ import { KioskState } from "./KioskState";
 import configData from "../config.json";
 import { getGameDetailsAsync } from "../BackendRequests"
 import { tickEvent } from "../browserUtils";
+import { getSharedKioskData } from "../share";
+
+export interface KioskOpts {
```

Comment:
This is such a good change. I need to take advantage of interfaces and objects more often. Especially since we might be wanting to expand these options in the future. Thanks for doing this.